html/elements/audio.json

```diff
@@ -578,6 +578,3416 @@
               "deprecated": false
             }
           }
+        },
+        "3gp_container": {
```

Comment:
I'm thinking we should use something more distinctive for these, such as `media_container_3gp`.